### PR TITLE
Czech translation

### PR DESCRIPTION
**Pull request by Clonewayx**

### CODE REVIEW

File: app/src/main/res/values-cs/ 	strings.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    <string name="viewCardTitle">Zobrazit věrnostní kartu</string>
+    <string name="scanCardBarcode">Oskenujte kód karty</string>
+
+    <string name="barcodeImageDescription">Obrázek kódu karty</string>
+
+    <string name="noStoreError">Nebyl zadán Obchod</string>
+    <string name="noCardIdError">Nebylo zadáno ID karty</string>
+
+    <string name="cardIdFormat">%1$s: %2$s</string>
+    <string name="storeNameAndNoteFormat" translatable="false">%1$s - %2$s</string>
```
Comment (brarcher) [inline app/src/main/res/values-cs/ 	strings.xml]: You can remove this line from the translations, as it is not necessary. This string is marked as not translatable

**Comment by brarcher** [comment]
If, in the future, this application adds other strings that need translated will you be available and willing to maintain the Czech translations? If so, I'll gladly accept the contribution. (:

**Comment by Clonewayx** [comment]
OK

**Comment by brarcher** [comment]
Thanks!

**Pull request merged**